### ISSUE TITLE
Running multi_scan with custom_report_file_name generates hidden exception and does not build

### ISSUE THREAD

**Issue by lyndsey-ferguson** [lyndsey-ferguson/fastlane-plugin-test_center#73]
### New Issue Checklist

- [x] Updated `fastlane-plugin-test_center` to the latest version
- [x] I read the [README.md](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md)
- [x] I reviewed the [example(s)](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/README.md) for the action(s) I am using
- [x] I have removed any sensitive data such as passwords, authentication tokens, or anything else I do not want to world to see

### Issue Description

The example:
```
          UI.important(
            'example: ' \
            'multi_scan also works with just one test target in the Scheme.'
          )
          multi_scan(
            project: File.absolute_path('../AtomicBoy/AtomicBoy.xcodeproj'),
            scheme: 'Professor',
            try_count: 3,
            custom_report_file_name: 'atomic_report.xml',
            output_types: 'junit',
            fail_build: false
          )
```

Does not build because `scan` throws an exception. This happens because something is broken in `reportnamer` and `scan` is receiving a `custom_report_file_name` with multiple `output_types`. This prevents it from actually building. The exception is then hidden and does not bubble out to the user.

##### Complete output when running fastlane, including the stack trace and command used

#<FastlaneCore::Interface::FastlaneError: Using a :custom_report_file_name with multiple :output_types (html,junit) will lead to unexpected results. Use :output_files instead.>

**Comment by lyndsey-ferguson** [comment]
While I am changing `:custom_report_file_name` with `:output_files` to get around the problem in [this example](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/blob/master/lib/fastlane/plugin/test_center/actions/multi_scan.rb#L273), this really should be fixed.

**Comment by lyndsey-ferguson** [comment]
Fixed in 3.8.0.parallelizing.beta.8. Will close once it is released

**Comment by lyndsey-ferguson** [comment]
Fixed in [v3.8.0](https://github.com/lyndsey-ferguson/fastlane-plugin-test_center/releases/tag/v3.8.0)
